mern-todo/client/src/components/App.jsx

```diff
@@ -0,0 +1,96 @@
+import { Component } from 'react';
+import ReactDOM from 'react-dom';
+import ListIterator from 'Components/ListIterator';
+import FormComponent from 'Components/FormComponent';
+
+class App extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      items: [],
+    };
+    this.handleDelete = this.handleDelete.bind(this);
+    this.handleUserInput = this.handleUserInput.bind(this);
+  }
+
+  componentDidMount() {
+    axios.get('http://localhost:3000/todos')
```

Comment:
Just FYI, this is fine for practice purpose.
But normally we won't give full URLs here.
We will only give relative paths and handle the redirection through nginx via proxy passing

Reply:
Define the paths in a constant file